cmd/probe.go

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package cmd
+
+import (
+	"github.com/kubearmor/kubearmor-client/probe"
+	"github.com/kubearmor/kubearmor-client/deployment"
+	"github.com/spf13/cobra"
+	"github.com/fatih/color"
+	"fmt"
+	"bufio"
+	"os"
+)
+
+var probeInstallOptions deployment.ProbeOptions
+
+// probeCmd represents the get command
+var probeCmd = &cobra.Command{
+	Use:   "probe",
+	Short: "Display probe information",
+	Long:  `Display probe information`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		color.Blue("To get full probe, a pod will be deployed in your cluster - This pod will be deleted after probing")
```

Comment:
I was hoping for an Initial probe result, before we proceed with a full probe.
```suggestion
		color.Blue("To get full probe, a daemonset will be deployed in your cluster - This daemonset will be deleted after probing")
```